packages/accounts/wagmi.config.ts

```diff
@@ -24,11 +24,14 @@ export default defineConfig(
       plugingen({
         chain: config.chain,
         connectionConfig: {
-          rpcUrl: "https://ethereum-sepolia.publicnode.com",
+          rpcUrl:
+            config.rpcUrl ||
+            (config.chain.rpcUrls?.default?.http.length > 0
+              ? config.chain.rpcUrls.default.http[0]
+              : `https://${config.chain.name
+                  .toLowerCase()
+                  .replace("-testnet", "")}-rpc.publicnode.com`),
```

Comment:
this doesn't look right. for example the old `sepolia` one was `https://ethereum-sepolia.publicnode.com` without the `-rpc`

Reply:
no it's updated. now there is -rpc at the end

Reply:
https://www.publicnode.com/